src/types.ts

```diff
@@ -29,7 +43,6 @@ export interface ServerlessInstance {
       }
     },
     custom: {
-      vpc: VPCDiscovery | undefined, // Deprecated
```

Comment:
Should add a note to the README that support for this is being removed

Reply:
Other option could be to keep support for this for one more release

Reply:
I Will update the readme